resources/home/dnanexus/make_workbook.py

```
@@ -252,6 +252,18 @@ def get_hpo_obo(self):
                 f"HPO version in JSON {version} not found in config\n"
                 f"{self.config}"
             )
+        # Get termPresence from JSON
+        term_presence = self.wgs_data[
+                    "interpretation_request_data"
+                ]['json_request']["pedigree"][
+            "members"
+            ][0]["hpoTermList"][0]['termPresence']
+
+        # If termPresence is unknown, remove variant from workbook
+        for k, v in filter(lambda item: item[1].get(term_presence, 'unknown') != "unknown", \
```

Ruff (0.13.1)

Loop control variable `k` not used within loop body

(B007)